src/components/EnvironmentVariables/index.js

```diff
@@ -0,0 +1,427 @@
+      </div>
+      {displayProjectVars.length == 0 ? (
+        <>
+          <hr style={{ margin: "30px 0" }} />
+          <div style={{ textAlign: "center" }}>No Project variable set</div>
+        </>
+      ) : (
+        <>
+          <hr style={{ margin: "30px 0" }} />
+          <div className="header">
+            <label>Project Variables</label>
+            <Button
+              onClick={() => showPrjVarValue()}
+              aria-controls="example-collapse-text"
+              aria-expanded={openPrjVars}
+            >
+              {!openPrjVars ? "Show values" : "Hide values"}
+            </Button>
+          </div>
+          <div className="field-wrapper env-vars">
+            <Table striped bordered hover>
+              <thead>
+                <tr>
+                  <th>Project Variable Name</th>
+                  <th>Project Variable Scope</th>
+                  <Collapse in={openPrjVars}>
+                    <th>Value</th>
+                  </Collapse>
+                </tr>
+              </thead>
+              <tbody>
+                {displayProjectVars.map((projEnvVar, index) => {
+                  return (
+                    <Fragment key={index}>
+                      <tr>
+                        <td className="varName">{projEnvVar.name}</td>
+                        <td className="varScope">{projEnvVar.scope}</td>
+                        {prjLoading ? (
+                          <Collapse in={openPrjVars}>
+                            <td className="varValue" id={index}>
+                              <div className="loader"></div>
+                            </td>
+                          </Collapse>
+                        ) : projEnvVar.value ? (
+                          <Collapse in={openPrjVars}>
+                            <td className="varValue" id={index}>
+                              {projEnvVar.value.length <= 100 &&
+                              !prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(projEnvVar.value).substring(0, 25)}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesShow(index)}
+                                  >
+                                    <Image
+                                      src={show}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length <= 100 &&
+                                prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {projEnvVar.value}
+                                  <a
```

Comment:
same with using spans instead or you could actually add the click handler to the image